core/src/main/java/io/neow3j/transaction/TransactionBuilder.java

```diff
@@ -389,19 +363,18 @@ private long getSystemFeeForScript() throws IOException {
     private long calcNetworkFee() throws IOException {
```

Comment:
Two exceptions thrown in this method are out of date and still contain references to a wallet. Please adapt them. 
(I couldn't mark the code locations because there not affected by any existing changes.)

Reply:
done